docs/pages/server-access/guides.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Server Access Guides
+description: How to set up and configure Teleport Server Access for SSH and PAM
+---
+
+# Guides
+
+- [SSH](./guides/teleport-ssh.mdx)
```

Comment:
Try to use tile image group here:

https://teleport-docs-next-git-feature-partners-block-gravitational.vercel.app/storybook/?path=/story/docs-tile-tileimage--tile-image-group

With a new layout:

```
---
title: Server Access Guides
description: How to set up and configure Teleport Server Access for SSH and PAM
layout: tocless-doc
---